docs/index.md

```diff
@@ -418,6 +418,30 @@ nickname or tablename, that object is the target reference.
 Otherwise, the reference can be to an object that has not been created yet. Snowfakery will generate an ID for the object so that the current row can be generated. No other properties of the other object can be referred to, because
 it does not exist yet.
 
+If the referenced name contains dots then Snowfakery will follow
+field names to get to the final name. The first name before the first "." could be either a field name or an object-name.
```

Comment:
Are we okay with changing behavior if someone was already using dots in names?

Reply:
Snowfakery did always use a Python-meaning for dots in Jinja. Also I don't know of any users using Snowfakery outside of the SalesForce universe. But I'll note it as a potential backwards incompatibility.